libcloud/compute/drivers/gce.py

```diff
@@ -7073,9 +7073,10 @@ def ex_get_subnetwork(self, name, region=None):
         """
         region_name = None
         if name.startswith('https://'):
-            parts = self._get_components_from_path(name)
-            name = parts['name']
-            region_name = parts['region']
+            #parts = self._get_components_from_path(name)
+            #name = parts['name']
+            #region_name = parts['region']
```

Comment:
Some initial feedback: please remove code instead of commenting it. Same comment for `# request = '/regions/%s/subnetworks/%s' % (region_name, name)` below. Thanks!

Reply:
done!